content/projects/gsoc.adoc

```diff
@@ -4,123 +4,197 @@ title: "Google Summer of Code"
 
-* link:https://github.com/rtyler[R Tyler Croy]
-* link:https://github.com/batmat[Baptiste Mathaus]
+| link:https://wiki.jenkins-ci.org/display/JENKINS/Support+Core+Plugin[Support core plugin] improvements 
+| Plugins
+| It is often difficult for plugins developers to diagnose issues and analyse the user environment.
 
-=== External Workspace Manager Plugin
+The link:https://wiki.jenkins-ci.org/display/JENKINS/Support+Core+Plugin[Support Core plugin] allows users to generate a bundle to help on this but it is nowadays rarely used because it is isn't user friendly.
+Various fixes and improvements may help a lot our community. 
 
-link:https://gitter.im/jenkinsci/external-workspace-manager-plugin?utm_source=share-link&utm_medium=link&utm_campaign=share-link[image:https://badges.gitter.im/jenkinsci/external-workspace-manager-plugin.svg[title: "Gitter"]]
+Few ideas of new features to add:
 
-Currently, Jenkins’ build workspace may become very large in size due to the
-fact that some compilers generate very large volumes of data. The existing
-plugins that share the workspace across builds are able to do this by copying
-the files from one workspace to another, process which is inefficient.
+* Ease the management of bundles (UI to list, delete, browse, download bundles) - JENKINS-33090
+* Bundles will have to be stored in our public JIRA thus it is critical to provide anonymisation of the content by default - JENKINS-21670
+* Submission of bundles in link:https://issues.jenkins-ci.org[our bug tracker] - JENKINS-33091
+* etc.
 
-A solution for this problem is to create the
-link:https://github.com/jenkinsci/external-workspace-manager-plugin[External
-Workspace Manager Plugin]. The plugin should:
+Prerequisites:
 
-* provide an external workspace management system
-* facilitate workspace share and reuse across multiple Jenkins jobs
-* eliminate the need to copy, archive or move files
+* Basic knowledge of Jenkins (as a user)
+* Basic knowledge of Java programming language
 
-Additional details about the plugin may be found on the
-link:/blog/2016/05/23/external-workspace-manager-plugin/[blog post] introduction.
+| link:https://github.com/alecharp[Adrien Lecharpentier], link:https://github.com/christ66[Steven Christou]
+|======================
 
-==== Work summary
+=== Application process
 
-Here's a summary of the contributions that have been made during this project: https://alexsomai.github.io/gsoc-2016/.
+1. Check out the project ideas (see above).
+2. Select an interesting project idea or draft your own proposal.
+3. If you are not familiar with Jenkins, read the introductory info on the website and trying it out with one of your previous projects.
+4. Join the _jenkinsci-dev_ and _jenkinsci-gsoc-all-public_ mailing lists
+5. If you do not have a GitHub account, create one
+6. Introduce yourself the community and start your project proposal discussion by sending an introductory email to the _jenkinsci-dev_ Google Group (see the guidelines below).
```

Comment:
Perhaps link _jenkins-dev_ to https://groups.google.com/forum/#!forum/jenkinsci-dev